Makefile

```diff
@@ -10,27 +11,39 @@ datarootdir = $(prefix)/share
 mandir = $(prefix)/share/man
 man1dir = $(mandir)/man1
 
-SOURCES = minecraftd.sh.in minecraftd.conf.in minecraftd.service.in minecraftd.sysusers.in minecraftd.tmpfiles.in minecraftd-backup.service.in minecraftd-backup.timer.in
+SOURCES = $(wildcard *.in)
 OBJECTS = $(SOURCES:.in=)
 
 GAME = minecraft
 INAME = minecraftd
 SERVER_ROOT = /srv/$(GAME)
+CONFIG_PATH = /etc/conf.d/$(GAME)
+SYSCONFDIR = /etc
+INSTANCE_CONFIG_DIR = $(SYSCONFDIR)/$(GAME)
 BACKUP_DEST = $(SERVER_ROOT)/backup
+LIBRARY_PATH = $(libdir)/$(GAME)
 BACKUP_PATHS = world
 BACKUP_FLAGS = -z
 KEEP_BACKUPS = 10
 GAME_USER = $(GAME)
+SERVER_MEMORY_INITIAL = 512
+SERVER_MEMORY_MAXIMUM = 1024
```

Comment:
I'll need to look into this further. You mentioned cuberite which I am not familiar with. `-Xms` and `-Xmx` are arguments to the JVM executable so they should work regardless of what server implementation is being used. Can you help me understand what I'm missing?

Reply:
Yes all java based implementations should honor this flag or more precisely, it will be enforced by the JVM. However, [cuberite](https://github.com/cuberite/cuberite) is a C++ rewrite and does not care about any of this.